sdk/template/template-dpg/src/WidgetServiceClient.ts

```diff
@@ -25,6 +25,14 @@
  * If you need to make changes, please do so in the original source file, \{project-root\}/sources/custom
  */
 
+/**
```

Comment:
I hate that the customization tool is not smart enough to not duplicate this.

Reply:
oops yes I have an issue logged https://github.com/Azure/azure-sdk-for-js/issues/26741